Docker/docker-compose.yml

```diff
@@ -11,109 +11,136 @@ services:
             - var-run-postgresql:/var/run/postgresql
     # Once off container to run the DB migrations
     migrate:
-        build: ./nab
+        build:
+            context: ..
+            dockerfile: Docker/nab/Dockerfile
         volumes:
             - ../:/home/pi/pynab
             - var-run-postgresql:/var/run/postgresql
         depends_on:
             - db
         command: ["/bin/bash", "/usr/local/bin/run-migrate.sh"]
-    nabdevd:
-        image: registry.gitlab.com/nguillaumin/nabdevd:latest
+    nabd:
+        build:
+            context: ..
+            dockerfile: Docker/nab/Dockerfile
+        volumes:
+            - ../:/home/pi/pynab
+            - var-run-postgresql:/var/run/postgresql
+        command: ["/bin/sh", "-c", "ln -sf /dev/stdout /var/log/nabd && /home/pi/venv/bin/python3 -m nabd.nabd"]
         ports:
-            - 3000:3000
+            - 10543:10543
+            - 10544:10544
     nabweb:
-        build: ./nab
+        build:
+            context: ..
+            dockerfile: Docker/nab/Dockerfile
         volumes:
             - ../:/home/pi/pynab
             - var-run-postgresql:/var/run/postgresql
         depends_on:
             - db
             - migrate
-            - nabdevd
+            - nabd
         command: ["/bin/sh", "-c", "/home/pi/venv/bin/gunicorn --timeout 60 -b 0.0.0.0 nabweb.wsgi"]
```

Comment:
Maybe here we need to wait for nabd as well? That's perhaps why the web interface fails to connect to it, if it's up before nabd has fully started.

Reply:
The issue was with nabio_virtual's implementation of gestalt, which wasn't async. Hence the timeout.